adapters/webui/internal/frontend/home.go

```diff
@@ -0,0 +1,38 @@
+package frontend
+
+import (
+	"embed"
+	"html/template"
+	"net/http"
+)
+
+// Embed the template file
+//
+//go:embed home.html
+var templateFS embed.FS
```

Comment:
if you only have a single file I think you should embed it like so
```
//go:embed home.html
var homeTemplate string
```
then use
```
t, err := template.New("home").Parse(homeHTML)
```

Reply:
if you have a separate js file then i think i would still use two separate string embeds because it provides a compile error for missing files

Reply:
Okay nice. Thanks for the suggestion! Will try move the js out to a seperate file then add as a parameter.

Reply:
Seems like I can do it if its type template.JS

Reply:
Great so all I had to do was parse the js file (as a string) and execute it using the html/template package to render out the paths. The execute was written to a string and then provided to the next template as a parameter where the JS is injected as a template.JS so that its correctly handled as documented here: https://github.com/golang/go/blob/master/src/html/template/content.go